Fixes #324: More examples for new tutorial

andy-maier opened the pull request:
This PR now covers as much as the existing tutorial page on github.io coveres.

If we agree that creating new tutorials for the remaining functionality can be left to the future, **this PR is ready for review and merge**.

Note that the links to the online notebook viewer will only work once this PR has been merged into master. And the best way probably to review the notebooks is to see them in that viewer (unless you use the new `tools/jnb` script of course and view them in your local jupyter notebook invocation).

Details from the commit log:

* Added Jupyter Notebook files for all sections of the existing tutorial page on `pywbem.github.io`. As a result, not all operations are covered, but those that were covered befopre, or that had examples are now covered. Each Jupyter Notebook is a tutorial.
* Added a tutorial chapter to the RTD documentation, that explains how to use the Jupyter Notebooks, and that links to the tutorials (i.e. the notebook files). The links open the notebook files in the online notebook viewer at `nbviewer.jupyter.org`.
* Added a shell script `tools/jnb` that starts Jupyter Notebook locally, for the developers of such notebooks, and for users that want to run or modufy the notebooks. If needed, the script also sets up the environment by creating a Python virtual environment and by installing jupyter into it.
* Updated the makefile to add notebook files (with wildcards) to documentation dependents. Changed .rst files in dependents to become wildcard-based as well.

KSchopmeyer commented:
I suggest that we not use CIM_ComputerSystem as the delete instance example.  That can play havoc with a system if anybody tries to play with that toy.

Maybe we can tie the create instance and delete instance together for the same object.

I wish we had some play Classes but right now Pegasus does but not anybody else.

KSchopmeyer commented:
Other than that comment I will +1 this.  We may change after tests.

andy-maier commented:
Addressed your comment by merging the create and delete tutorials so that we are now creating and deleting a dynamic filter instance.
Rebased.
Merging.
